src/app/components/Link/index.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '../../helpers/tests/testHelpers';
+import Link from './index';
+
+const text = 'Some link text';
+const href = 'https://www.bbc.com/';
+
+describe('Link', () => {
+  shouldMatchSnapshot(
+    'should render a standard block link',
+    <Link text={text} href={href} />,
+  );
+  shouldMatchSnapshot(
+    'should render an inline link after passing an inline prop',
+    <Link text={text} href={href} inline />,
```

Comment:
Could you explain why this isn't `inline=true` here?

Reply:
Inline link will not not have display = block. So that it stays on the same line as the rest of the content. Also if I assign inline={true} prettifier will complain.

Reply:
Thanks for the explanation - another thing to list that I don't agree with prettier on 🙊